spec/mind_match_spec.rb

```diff
@@ -36,13 +37,42 @@
 
     it 'returns an id for a list of talents & position' do
       VCR.use_cassette("create_match") do
-        expect(mindmatch.create_match(talents: talents, position: position)).to eql('7e117fde-eb2c-423b-818e-86447b7049f3')
+        expect(mindmatch.create_match(talents: talents, position: position)).to eql('1d7964c9-7219-4d77-aee3-ece7919f9af5')
       end
     end
 
-    it 'returns a score for a match id' do
+    let(:match) { mindmatch.query_match(id: '6ecc0537-63ea-4a39-9d4c-e2c2b054b70d') }
+
+    it 'returns the status for a match id' do
+      VCR.use_cassette("query_match") do
+        expect(match["id"]).to eql('53b03dc2-07dd-40a2-91fe-7bff24c86574')
+        expect(match["status"]).to eql('fulfilled')
+      end
+    end
+
+    let(:match_result) { match.dig("data", "results")[0] }
+
+    it 'returns the score and the ids for a match id' do
+      VCR.use_cassette("query_match") do
+        expect(match_result["score"]).to eql(0.1436655436)
+        expect(match_result["personId"]).to eql('7dedffb3-c41e-4046-aa3d-73979d7ec1c2')
+        expect(match_result["positionId"]).to eql('6214ab8d26e3f79571d922ca269d5743')
+      end
+    end
+
+    let(:match_people) { match.dig("data", "people")[0] }
```

Comment:
the mindmatch api will remove the "data" namespace key in the future
that's why we had that line https://github.com/mindmatch/mindmatch-ruby/pull/1/files#diff-fa5a46447ed615900fc67e83a69af91fL62

so that the response would not include it and would work with current and upcoming version of the api that does not include the namespace

something like 

`match&.has_key?('data')`

would probably be enough

Reply:
>the mindmatch api will remove the "data" namespace key in the future

But at that time we'll update the gem and the cassettes so that it is up to date with the MM APIs, won't we?